src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
We can improve readability by split this code into parts, e.g.:

`const filteredMen = people.filter((p) => p.sex === 'm');`
  `if (century) {`
   `filteredMen.forEach((m) => m.century = Math.ceil(m.died / 100));`
    `filteredMen = men.filter((m) => m.century === century);`
  `}`